content/foundation/governing-board/index.md

```diff
@@ -0,0 +1,121 @@
+
+There are nine different constituency groups that are allocated seats on the Governing Board, across three categories:
+
+* **Community representatives**
+  * 4 seats for Individual Members – these are people who currently provide financial support to the Foundation.
+  * 3 seats for Ecosystem Members – these are FOSS projects and communities around Matrix.
+  * 2 seats for Associate Members – these are FOSS foundations, academic institutions, and other communities that are aligned with key parts of our mission.
+* **Funder representatives**
+  * 4 seats for Platinum Members – these organisations sponsor at the highest level.
+  * 3 seats for Gold Members — these organisations sponsor at a high level.
+  * 2 seats for Silver Members — these organisations sponsor at a level commensurate with the size of their organisation.
+* **Foundation representatives**
+  * 3 seats for Guardians – these are the members of the Foundation's board of directors.
+  * 2 seats for the Spec Core Team – these are the members of the Spec Core Team that looks after the spec.
+  * Managing Director – this is the Foundation's head of staff.
+
+Elections are being held in the second quarter of each calendar year, alternating halves of the Constituency Groups representatives on the Governing Board as per the following calendar:
+
+* April-June 2024, and any subsequent even-numbered years: Guardians, SCT, Platinum and Ecosystem
+* April-June 2025, and any subsequent odd-numbered years: Gold, Silver, Individual, Associate
+
+## Elected representatives
+
+The current elected representatives are:
+
+<div class="two-column">
+<div>
+
+### Individual Members
+
+* Andy Balaam (he/him)
+* Greg Sutcliffe (he/him)
+* J.B. Crawford (they/them)
+* Sumner Evans
+
+</div>
+<div>
+
+### Ecosystem Members
+
+* Bram van den Heuvel (he/they), Elm SDK
+* Kim Brose (he/him), Matrix Community Events
+* Nicolas Werner (he/him), Nheko-Reborn
+
+</div>
+<div>
+
+### Associate Members
+
+* Cleo Menenez Jr. (he/him), GNOME Foundation
+* Tobias Fella (he/him), KDE e.V.
+
+</div>
+<div>
+
+### Platinum Members
+
+* Neil Johnson (he/him), Element
+
+</div>
+<div>
+
+### Gold Members
+
+* Brad Murray (he/him), Automattic (Beeper)
+* Kevin Boos (he/him), Futurewei Technologies
+
+</div>
+<div>
+
+### Silver Members
+
+* Jan Kohnert (he/him), Gematik GmbH
+* Thor Arne Johansen, Verji Tech AS
+
+</div>
+<div>
+
+### Guardians
+
+* Amandine Le Pape (she/her)
+* Matthew Hodgson (he/him)
+* Ross Schulman
+
+</div>
+<div>
+
+### Spec Core Team
+
+* Richard van der Hoff (he/him)
+* Travis Ralston (he/him)
+
+</div>
+
+</div>
+
+## Where can I reach the Governing Board?
+
+The Governing Board has a dedicated room on Matrix, [#governing-board-office:matrix.org](https://matrix.to/#/#governing-board-office:matrix.org), where you can reach out to them.
+
+## Committees
+
+There are currently 4 Committees:
+
+* Community
+* Governance
+* Trust & Safety
+* Finance & Fundraising
```

Comment:
@GregSutcliffe I expect we will want to add a quick sentence about each. although their charters are not all finalized (but should probably eventually also be published), the rough direction is clear, no?

we also have chair and vice for each.

Reply:
Yes, we'll want to add that, though the names are fairly clear. Lets not block this, we can follow up after the charters are done